shell/platform/android/test/io/flutter/plugin/editing/TextEditingDeltaTest.java

```diff
@@ -0,0 +1,56 @@
+package io.flutter.plugin.editing;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.MockitoAnnotations;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+@Config(manifest = Config.NONE)
+@RunWith(RobolectricTestRunner.class)
+public class TextEditingDeltaTest {
+  @Before
+  public void setUp() {
+    MockitoAnnotations.initMocks(this);
+  }
+
+  @Test
+  public void testConstructorTextEditingDelta() {
+    final CharSequence oldText = "hell";
+    final CharSequence textAfterChange = "hello";
+
+    final int oldComposingStart = 0;
+    final int oldComposingEnd = 4;
+
+    final int startOfReplacementText = 0;
+    final int endOfReplacementText = textAfterChange.length();
+
+    final int newSelectionStart = 5;
+    final int newSelectionEnd = 5;
+    final int newComposingStart = 0;
+    final int newComposingEnd = 5;
+
+    final TextEditingDelta delta =
+        new TextEditingDelta(
+            oldText,
+            oldComposingStart,
+            oldComposingEnd,
+            textAfterChange,
+            newSelectionStart,
+            newSelectionEnd,
+            newComposingStart,
+            newComposingEnd);
+
+    assertEquals(oldText, delta.getOldText());
+    assertEquals("hello", delta.getDeltaText());
```

Comment:
Why the string literal instead of `textAfterChange`?

Reply:
Updated